bin/README.md

```diff
@@ -0,0 +1,23 @@
+# arduino101load
+multiplatform launcher for Arduino101 dfu-util flashing utility
+
+## Compiling
+
+* Download go package from [here](https://golang.org/dl/) or using your package manager
+* `cd` into the root folder of this project
```

Comment:
Well, it's not the root directory anymore... I had to cd into the "bin"directory to get the build to work